### PR TITLE
Go-to-market phase name updates

### PR DESCRIPTION
**Pull request by lnhsingh**
General cleanup changes (that don't need a review):
- Removed the `private-preview` include, as that will not be a publicly documented GTM phase.
- Changed `preview-opt-in` include to be named `limited-access` and updated the language. Closes DOC-6966
- Removed liquid from the `preview` and `limited-access` includes so they can be used more flexibly. Added liquid around the `preview` and `limited-access` included content in the docs where the includes are used. Closes DOC-6242.
- Updated old "experimental" terminology to "preview" for `--accept-sql-without-tls` (DOC-5578)
- Updated old "experimental" terminology to "preview" for temporary objects (DOC-5398)
- Updated old "experimental" terminology to "preview" for `EXPERIMENTAL SCRUB` (DOC-5591)
- Renamed the Features in Preview page to be more generalized "CockroachDB Feature Availability"
- Removed strptime/strftime builtins from the list of preview features in v22.2 and v23.1. Closes DOC-5780 and DOC-5397.
- Removed CDC External Connections from the list of preview features in v23.1.

**Larger changes (please review):**
- Renamed the Features in Preview page to be more generalized "[CockroachDB Feature Availability](https://deploy-preview-16291--cockroachdb-docs.netlify.app/docs/stable/cockroachdb-feature-availability.html)" and relocated it in the sidenav
- Added the following net new sections to CockroachDB Feature Availability page:
    - [Feature availability phases](https://deploy-preview-16291--cockroachdb-docs.netlify.app/docs/stable/cockroachdb-feature-availability.html#feature-availability-phases). Closes DOC-5013.
    - [Features in limited access](https://deploy-preview-16291--cockroachdb-docs.netlify.app/docs/stable/cockroachdb-feature-availability.html#features-in-limited-access)
    - [Google Pub/Sub](https://deploy-preview-16291--cockroachdb-docs.netlify.app/docs/stable/cockroachdb-feature-availability.html#google-pub-sub-sink-for-changefeeds)
    - [Webhook sink](https://deploy-preview-16291--cockroachdb-docs.netlify.app/docs/stable/cockroachdb-feature-availability.html#webhook-sink-for-changefeeds)
    - [CDC transformations](https://deploy-preview-16291--cockroachdb-docs.netlify.app/docs/stable/cockroachdb-feature-availability.html#change-data-capture-transformations)
    - [External connections](https://deploy-preview-16291--cockroachdb-docs.netlify.app/docs/stable/cockroachdb-feature-availability.html#external-connections)

### CODE REVIEW

File: v22.1/cockroach-start-single-node.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Flag | Description
...
-`--accept-sql-without-tls` | This experimental flag allows you to connect to the cluster using a SQL user's password without [validating the client's certificate](authentication.html#client-authentication). When connecting using the built-in SQL client, [use the `--insecure` flag with the `cockroach sql` command](cockroach-sql.html#client-connection).
+`--accept-sql-without-tls` | This flag (in preview) allows you to connect to the cluster using a SQL user's password without [validating the client's certificate](authentication.html#client-authentication). When connecting using the built-in SQL client, [use the `--insecure` flag with the `cockroach sql` command](cockroach-sql.html#client-connection).
```
Comment (shannonbradshaw) [inline v22.1/cockroach-start-single-node.md:69]: I think it would be clearer to include a note stating that this flag is in preview. As a reader, I'm unsure how to interpret "(in preview)". Could link "preview" to cockroachdb-feature-availability.md. This comment applies to other places where we're doing something similar in this PR.

**Review by shannonbradshaw — APPROVED** [review]
One comment, but it applies in several places. Otherwise, LGTM.

File: v22.2/cockroachdb-feature-availability.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
+---
+title: CockroachDB Feature Availability
+summary: Learn about the features available in preview and limited access in CockroachDB
+toc: true
+docs_area: reference.sql
+key: experimental-features.html
+---
+
+CockroachDB features go through a development lifecycle and some are publicly available at different phases than others. This page defines the different levels of CockroachDB {{ page.version.version }} feature availability, and lists the features in each phase.
```
Comment (nickvigilante) [inline v22.2/cockroachdb-feature-availability.md:9]: Nit

```suggestion
CockroachDB features go through a development lifecycle, and some are publicly available at different phases than others. This page defines the different levels of CockroachDB {{ page.version.version }} feature availability and lists the features in each phase.
```

File: v22.2/cockroachdb-feature-availability.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+### Egress perimeter controls for {{ site.data.products.dedicated }}
+
+[Egress Perimeter Controls](../cockroachcloud/egress-perimeter-controls.html) can enhance the security of {{ site.data.products.dedicated }} clusters by enabling cluster administrators to restrict egress to a list of specified external destinations. This adds a strong layer of protection against malicious or accidental data exfiltration.
+
+### Private {{ site.data.products.dedicated }} clusters
+
+Limiting access to a CockroachDB cluster's nodes over the public internet is an important security practice and is also a compliance requirement for many organizations. [{{ site.data.products.dedicated }} private clusters](../cockroachcloud/private-clusters.html) allow organizations to meet this objective. A private {{ site.data.products.dedicated }} cluster's nodes have no public IP addresses, and egress traffic moves over private subnets and through a highly-available NAT gateway that is unique to the cluster
```
Comment (nickvigilante) [inline v22.2/cockroachdb-feature-availability.md:40]:
```suggestion
Limiting access to a CockroachDB cluster's nodes over the public internet is an important security practice and is also a compliance requirement for many organizations. [{{ site.data.products.dedicated }} private clusters](../cockroachcloud/private-clusters.html) allow organizations to meet this objective. A private {{ site.data.products.dedicated }} cluster's nodes have no public IP addresses, and egress traffic moves over private subnets and through a highly-available NAT gateway that is unique to the cluster.
```

File: v22.2/cockroachdb-feature-availability.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+## Features in limited access
+
+{{site.data.alerts.callout_info}}
+**The following features are in limited access** and are only available to enrolled organizations. To enroll your organization, contact your Cockroach Labs account team. These features are subject to change.
+{{site.data.alerts.end}}
+
+### Export logs from {{ site.data.products.dedicated }} clusters
+
+{{ site.data.products.dedicated }} users can use the [Cloud API](../cockroachcloud/cloud-api.html) to configure [log export](../cockroachcloud/export-logs.html) to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [GCP Cloud Logging](https://cloud.google.com/logging). Once the export is configured, logs will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud log sink. You can configure log export to redact sensitive log entries, limit log output by severity, send log entries to specific log group targets by log channel, among others.
```
Comment (nickvigilante) [inline v22.2/cockroachdb-feature-availability.md:28]:
```suggestion
{{ site.data.products.dedicated }} users can use the [Cloud API](../cockroachcloud/cloud-api.html) to configure [log export](../cockroachcloud/export-logs.html) to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [GCP Cloud Logging](https://cloud.google.com/logging). Once the export is configured, logs will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud log sink. You can configure log export to redact sensitive log entries, limit log output by severity, and send log entries to specific log group targets by log channel, among others.
```

File: v23.1/cockroachdb-feature-availability.md [hunk 5]
```diff
@@ -1,23 +1,53 @@
...
+## Features in limited access
 
-## SQL statements
+{{site.data.alerts.callout_info}}
+**The following features are in limited access** and are only available to enrolled organizations. To enroll your organization, contact your Cockroach Labs account team. These features are subject to change.
+{{site.data.alerts.end}}
+
+### Export logs from {{ site.data.products.dedicated }} clusters
+
+{{ site.data.products.dedicated }} users can use the [Cloud API](../cockroachcloud/cloud-api.html) to configure [log export](../cockroachcloud/export-logs.html) to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [GCP Cloud Logging](https://cloud.google.com/logging). Once the export is configured, logs will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud log sink. You can configure log export to redact sensitive log entries, limit log output by severity, send log entries to specific log group targets by log channel, among others.
```
Comment (nickvigilante) [inline v23.1/cockroachdb-feature-availability.md:28]:
```suggestion
{{ site.data.products.dedicated }} users can use the [Cloud API](../cockroachcloud/cloud-api.html) to configure [log export](../cockroachcloud/export-logs.html) to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [GCP Cloud Logging](https://cloud.google.com/logging). Once the export is configured, logs will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud log sink. You can configure log export to redact sensitive log entries, limit log output by severity, and send log entries to specific log group targets by log channel, among others.
```

File: v23.1/cockroachdb-feature-availability.md [hunk 6]
```diff
@@ -1,23 +1,53 @@
...
+
+### Egress perimeter controls for {{ site.data.products.dedicated }}
+
+[Egress Perimeter Controls](../cockroachcloud/egress-perimeter-controls.html) can enhance the security of {{ site.data.products.dedicated }} clusters by enabling cluster administrators to restrict egress to a list of specified external destinations. This adds a strong layer of protection against malicious or accidental data exfiltration.
+
+### Private {{ site.data.products.dedicated }} clusters
+
+Limiting access to a CockroachDB cluster's nodes over the public internet is an important security practice and is also a compliance requirement for many organizations. [{{ site.data.products.dedicated }} private clusters](../cockroachcloud/private-clusters.html) allow organizations to meet this objective. A private {{ site.data.products.dedicated }} cluster's nodes have no public IP addresses, and egress traffic moves over private subnets and through a highly-available NAT gateway that is unique to the cluster
```
Comment (nickvigilante) [inline v23.1/cockroachdb-feature-availability.md:40]:
```suggestion
Limiting access to a CockroachDB cluster's nodes over the public internet is an important security practice and is also a compliance requirement for many organizations. [{{ site.data.products.dedicated }} private clusters](../cockroachcloud/private-clusters.html) allow organizations to meet this objective. A private {{ site.data.products.dedicated }} cluster's nodes have no public IP addresses, and egress traffic moves over private subnets and through a highly-available NAT gateway that is unique to the cluster.
```

File: v23.1/cockroachdb-feature-availability.md [hunk 7]
```diff
@@ -1,23 +1,53 @@
...
-summary: Learn about the features available in preview in CockroachDB
+title: CockroachDB Feature Availability
+summary: Learn about the features available in preview and limited access in CockroachDB
 toc: true
 docs_area: reference.sql
 key: experimental-features.html
 ---
 
-This page lists the features that are available in preview in CockroachDB {{ page.version.version }}. These features are subject to change. To share feedback and/or issues, contact [Support](https://support.cockroachlabs.com/hc/en-us).
+CockroachDB features go through a development lifecycle and some are publicly available at different phases than others. This page defines the different levels of CockroachDB {{ page.version.version }} feature availability, and lists the features in each phase.
```
Comment (nickvigilante) [inline v23.1/cockroachdb-feature-availability.md:9]: Nit

```suggestion
CockroachDB features go through a development lifecycle, and some are publicly available at different phases than others. This page defines the different levels of CockroachDB {{ page.version.version }} feature availability and lists the features in each phase.
```

**Review by nickvigilante — APPROVED** [review]
A few nits, but LGTM otherwise!

File: v23.1/cockroachdb-feature-availability.md [hunk 8]
```diff
@@ -137,12 +159,28 @@ $ cockroach sql --user=jpointsman --insecure
...
 ~~~
 
-## Changefeed metrics labels
+### Core implementation of changefeeds
+
+The [`EXPERIMENTAL CHANGEFEED FOR`](changefeed-for.html) statement creates a new core changefeed, which streams row-level changes to the client indefinitely until the underlying connection is closed or the changefeed is canceled. A core changefeed can watch one table or multiple tables in a comma-separated list.
+
+### Changefeed metrics labels
 
 {% include {{ page.version.version }}/cdc/metrics-labels.md %}
```
Comment (mikeCRL) [inline v23.1/cockroachdb-feature-availability.md:168]: Feel free to save for a follow-up (that maybe Kathryn can take) as this was already here, but FWIW, this include provides a lot of content here. May be worth replacing it with the first paragraph from the description, and the key 'not available in cloud' bullet.

```suggestion
To measure metrics per changefeed, define a "metrics label" to which one or multiple changefeed(s) will increment each [changefeed metric](monitor-and-debug-changefeeds.html#metrics). Metrics label information is sent with time-series metrics to `http://{host}:{http-port}/_status/vars`, viewable via the [Prometheus endpoint](monitoring-and-alerting.html#prometheus-endpoint). An aggregated metric of all changefeeds is also measured.

Metrics labels are **not** available in {{ site.data.products.db }}```

Regardless that first sentence is hard to parse - FYI @kathancox.

File: v23.1/cockroachdb-feature-availability.md [hunk 9]
```diff
@@ -1,23 +1,53 @@
...
-| `enable_experimental_alter_column_type_general`       | `'false'`       |  If set to `'true'`, enables [column type altering](#alter-column-types) for general cases, with some limitations.                                                                                                                                                                                   |
-| `experimental_enable_temp_tables`       | `'off'`       |  If set to `'on'`, enables [temporary objects](#temporary-objects), including [temporary tables](temporary-tables.html), [temporary views](views.html#temporary-views), and [temporary sequences](create-sequence.html#temporary-sequences).                                                                                                                                                                                   |
+## Features in limited access
```
Comment (mikeCRL) [inline v23.1/cockroachdb-feature-availability.md:20]: FYI I was initially tempted to suggest that we convert the subheadings in these to a table of features, however, I like that this format offers a place to document features that don't yet have a home. We may want to follow up to confirm that the current state of docs is OK for these. Regardless, it's better than having the info hidden exclusively in release notes!

File: v22.1/cockroachdb-feature-availability.md [hunk 10]
```diff
@@ -1,27 +1,57 @@
...
+### Export logs from {{ site.data.products.dedicated }} clusters
+
+{{ site.data.products.dedicated }} users can use the [Cloud API](../cockroachcloud/cloud-api.html) to configure [log export](../cockroachcloud/export-logs.html) to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [GCP Cloud Logging](https://cloud.google.com/logging). Once the export is configured, logs will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud log sink. You can configure log export to redact sensitive log entries, limit log output by severity, send log entries to specific log group targets by log channel, among others.
+
+## Features in preview
+
 This page lists the features that are available in preview in CockroachDB {{ page.version.version }}. These features are subject to change. To share feedback and/or issues, contact [Support](https://support.cockroachlabs.com/hc/en-us).
 
-## Session variables
+### Export metrics from {{ site.data.products.dedicated }} clusters
```
Comment (mikeCRL) [inline v22.1/cockroachdb-feature-availability.md:34]: This is included twice. Also it's listed here as Preview but on the "metrics export"/export-metrics.html page, it's listed as Limited Access.

File: v22.1/cockroachdb-feature-availability.md [hunk 11]
```diff
@@ -1,27 +1,57 @@
...
+
+{{ site.data.products.dedicated }} users can use the [Cloud API](../cockroachcloud/cloud-api.html) to configure [metrics export](../cockroachcloud/export-metrics.html) to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [Datadog](https://www.datadoghq.com/). Once the export is configured, metrics will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud metrics sink.
+
+### Functions and Operators
```
Comment (mikeCRL) [inline v22.1/cockroachdb-feature-availability.md:38]: Since this section isn't for a single feature, the `Features in preview` section might flow better if this were the first or last subheading within.

File: v22.1/cockroachdb-feature-availability.md [hunk 12]
```diff
@@ -98,25 +128,17 @@ This example uses the `users` table from our open-source, fictional peer-to-peer
...
+### Temporary objects
 
 [Temporary tables](temporary-tables.html), [temporary views](views.html#temporary-views), and [temporary sequences](create-sequence.html#temporary-sequences) are in preview in CockroachDB. If you create too many temporary objects in a session, the performance of DDL operations will degrade. Performance limitations could persist long after creating the temporary objects. For more details, see [cockroachdb/cockroach#46260](https://github.com/cockroachdb/cockroach/issues/46260).
 
-## Password authentication without TLS
+To enable temporary objects, set the `experimental_enable_temp_tables` [session variable](show-vars.html) to `on`. 
+
+### Password authentication without TLS
 
 For deployments where transport security is already handled at the infrastructure level (e.g., IPSec with DMZ), and TLS-based transport security is not possible or not desirable, CockroachDB now supports delegating transport security to the infrastructure with the flag `--accept-sql-without-tls` (in preview) for [`cockroach start`](cockroach-start.html#security).
```
Comment (mikeCRL) [inline v22.1/cockroachdb-feature-availability.md:143]:
```suggestion
For deployments where transport security is already handled at the infrastructure level (e.g., IPSec with DMZ), and TLS-based transport security is not possible or not desirable, CockroachDB supports delegating transport security to the infrastructure with the flag `--accept-sql-without-tls` for [`cockroach start`](cockroach-start.html#security).
```

**Review by mikeCRL — CHANGES_REQUESTED** [review]
This is great. Left some suggestions. Just one apparent correction where the phase in the Feature Availability page doesn't match up with the feature's doc page.

File: cockroachcloud/cloud-api.md [hunk 13]
```diff
@@ -394,7 +394,9 @@ If the `DELETE` request was successful the client will not receive a response pa
 
 ## Export Cloud Organization audit logs
 
-{% include feature-phases/preview-opt-in.md %}
+{{site.data.alerts.callout_info}}
+{% include feature-phases/limited-access.md %}
```
[the remark this answers is not in the document]
Comment (lnhsingh) [reply cockroachcloud/cloud-api.md:398]: Done - thank you!

File: v23.1/cockroachdb-feature-availability.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (lnhsingh) [reply v23.1/cockroachdb-feature-availability.md:9]: This clarifies things, IMO. Changing.

File: v23.1/cockroachdb-feature-availability.md [hunk 14]
```diff
@@ -1,23 +1,53 @@
...
-## Session variables
+## Feature availability phases
 
-The table below lists the session settings that are available in preview. For a complete list of session variables, see [`SHOW {session variable}`](show-vars.html).
+Phase                                         | Definition | Accessibility 
+----------------------------------------------+------------+-------------
+Private preview                               | Feature is not production ready and will not be publicly documented. | Invite-only
+[Limited access](#features-in-limited-access) | Feature is production-ready but not available widely because of known limitations and/or because capabilities may change or be added based on feedback. | Feature is opt-in. To enroll your organization, contact your Cockroach Labs account team.
+[Preview](#features-in-preview)               | Feature is production-ready and publicly available. However, this feature may have known limitation and/or capabilities may change or be added based on feedback. | Public
+General availability (GA)                     | Feature is production-ready, publicly available, and has reached full development maturity. | Public
```
[the remark this answers is not in the document]
Comment (lnhsingh) [reply v23.1/cockroachdb-feature-availability.md:18]: I'm just going to remove this part

File: v23.1/cockroachdb-feature-availability.md [hunk 15]
```diff
@@ -1,23 +1,53 @@
...
-| `experimental_enable_temp_tables`       | `'off'`       |  If set to `'on'`, enables [temporary objects](#temporary-objects), including [temporary tables](temporary-tables.html), [temporary views](views.html#temporary-views), and [temporary sequences](create-sequence.html#temporary-sequences).                                                                                                                                                                                   |
+## Features in limited access
 
-## SQL statements
+{{site.data.alerts.callout_info}}
+**The following features are in limited access** and are only available to enrolled organizations. To enroll your organization, contact your Cockroach Labs account team. These features are subject to change.
+{{site.data.alerts.end}}
+
+### Export logs from {{ site.data.products.dedicated }} clusters
```
[the remark this answers is not in the document]
Comment (lnhsingh) [reply v23.1/cockroachdb-feature-availability.md:26]: This has actually been changed to be in preview now: https://www.cockroachlabs.com/docs/cockroachcloud/export-metrics.html (Andrew just updated and I haven't merged master lately)

File: v23.1/cockroachdb-feature-availability.md (hunk 9 above)
Comment (lnhsingh) [reply v23.1/cockroachdb-feature-availability.md:20]: I can see that working too. The previous structure of this page was with these headings and blurbs so I went with that.... but now thinking about it, maybe it is better to have them in a table and just have the table link out to the relevant docs pages.

That being said, there are at least two features (show table fingerprints and SCRUB) that don't have docs, so these blurbs are all we have (and wouldn't fit nicely into a table)

File: v22.1/cockroach-start-single-node.md (hunk 1 above)
Comment (lnhsingh) [reply v22.1/cockroach-start-single-node.md:69]: Good idea - I also forgot to add the link to the LA / Preview callouts.

File: v23.1/cockroachdb-feature-availability.md (hunk 8 above)
Comment (lnhsingh) [reply v23.1/cockroachdb-feature-availability.md:172]: Will open a new issue

Comment (mikeCRL) [reply v23.1/cockroachdb-feature-availability.md:172]: Thanks! (Saw the issue.)

File: v23.1/cockroachdb-feature-availability.md [hunk 16]
```diff
@@ -1,23 +1,57 @@
...
+
+### Egress perimeter controls for {{ site.data.products.dedicated }}
+
+[Egress Perimeter Controls](../cockroachcloud/egress-perimeter-controls.html) can enhance the security of {{ site.data.products.dedicated }} clusters by enabling cluster administrators to restrict egress to a list of specified external destinations. This adds a strong layer of protection against malicious or accidental data exfiltration.
+
+### Private {{ site.data.products.dedicated }} clusters
+
+Limiting access to a CockroachDB cluster's nodes over the public internet is an important security practice and is also a compliance requirement for many organizations. [{{ site.data.products.dedicated }} private clusters](../cockroachcloud/private-clusters.html) allow organizations to meet this objective. A private {{ site.data.products.dedicated }} cluster's nodes have no public IP addresses, and egress traffic moves over private subnets and through a highly-available NAT gateway that is unique to the cluster.
+
+### Export Cloud Organization audit logs (cloud API)
```
Comment (mikeCRL) [inline v23.1/cockroachdb-feature-availability.md:42]:
```suggestion
### Export Cloud Organization audit logs (Cloud API)
```

File: v22.2/cockroachdb-feature-availability.md [hunk 17]
```diff
@@ -0,0 +1,198 @@
...
+
+### Egress perimeter controls for {{ site.data.products.dedicated }}
+
+[Egress Perimeter Controls](../cockroachcloud/egress-perimeter-controls.html) can enhance the security of {{ site.data.products.dedicated }} clusters by enabling cluster administrators to restrict egress to a list of specified external destinations. This adds a strong layer of protection against malicious or accidental data exfiltration.
+
+### Private {{ site.data.products.dedicated }} clusters
+
+Limiting access to a CockroachDB cluster's nodes over the public internet is an important security practice and is also a compliance requirement for many organizations. [{{ site.data.products.dedicated }} private clusters](../cockroachcloud/private-clusters.html) allow organizations to meet this objective. A private {{ site.data.products.dedicated }} cluster's nodes have no public IP addresses, and egress traffic moves over private subnets and through a highly-available NAT gateway that is unique to the cluster.
+
+### Export Cloud Organization audit logs (cloud API)
```
Comment (mikeCRL) [inline v22.2/cockroachdb-feature-availability.md:42]:
```suggestion
### Export Cloud Organization audit logs (Cloud API)
```

**Review by mikeCRL — APPROVED** [review]
Looks great. Noticed one small thing I missed before—that "cloud API" should be capitalized in a heading on the Feature Availability page. Left it as a suggestion on both applicable copies (in the 22.2 and 23.1 docs).
